packages/parser-css/tests/basic-tests.ts

```diff
@@ -17,37 +20,35 @@ const element = {
     outerHTML(): Promise<string> {
         return Promise.resolve('');
     }
-};
+} as any;
 
-test.beforeEach((t) => {
-    t.context.element = element;
-    t.context.engine = new EventEmitter2({
+test('We should provide a correct AST when parsing CSS.', async (t) => {
+    const sandbox = sinon.createSandbox();
+    const engine = new EventEmitter2({
         delimiter: '::',
         maxListeners: 0,
         wildcard: true
-    });
-});
+    }) as Engine<StyleEvents>;
 
-test.serial('We should provide a correct AST when parsing CSS.', async (t) => {
-    const sandbox = sinon.createSandbox();
     const parseObject = {};
     const code = '.foo { color: #fff }';
     const style = `<style>  ${code}  </style>`;
-    new CSSParser.default(t.context.engine); // eslint-disable-line
+    new CSSParser.default(engine); // eslint-disable-line
+
+    const engineEmitASync = sandbox.spy(engine, 'emitAsync');
```

Comment:
Minor typo (also will need to update usages).
```suggestion
    const engineEmitAsync = sandbox.spy(engine, 'emitAsync');
```